src/locale/da.yml

```diff
@@ -129,7 +129,7 @@ feat:
         remove: Fjern fra opgave
       add:
         alreadyAdded: Allerede tilføjet
-        placeholder: Tilføj ringer
+        placeholder: Start med at skrive for at søge eller tilføje en ny opkalder
```

Comment:
someone should verify, weather this translation is accurate

Reply:
Nice work! But please remove all translations, as they are done by volunteers in an external system. These YAML files are generated and should (almost) never be modified manually.

Reply:
✅